guidelines.md

```diff
@@ -93,12 +106,12 @@ The authentication below will connect with the demo enterprise that is configure
 * **ClientToken** - `E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D`
 * **AccessToken** - `7059D2C25BF64EA681ACAB3A00B859CC-D91BFF2B1E3047A3E0DEC1D57BE1382`
 
-The enterprise is based in the United States, it accepts `GBP`, `EUR` and `USD` currencies (any of them may be used). As a tax rate, either `0.0`, `0.06`, `0.10`, `0.1025`, `0.1495` or `0.18` can be used.
+The enterprise is based in the United States, it accepts `GBP`, `EUR` and `USD` currencies (any of them may be used). 
 
 ### Production environment
 
 * **PlatformAddress** - `https://www.mews.li`
-* **ClientToken** - Unique token per integration. Will be provided to you by our integration team upon certification completion. For further information, please contact [marketplace@mewssystems.com](mailto://marketplace@mewssystems.com).
+* **ClientToken** - Unique token per integration. Will be provided to you by our integration team upon certification completion. For further information, please contact [marketplace@mews.com](mailto://marketplace@mews.com).
```

Comment:
For further information, please use the 'chat' on this page, not the marketplace email address

Reply:
updated text